bundle/manifests/lvm.topolvm.io_lvmclusters.yaml

```diff
@@ -301,10 +301,35 @@ spec:
                         properties:
                           devices:
                             description: Devices is the list of devices used by the
-                              deviceclass
+                              VG
                             items:
                               type: string
                             type: array
+                          excluded:
+                            description: Excluded contains the per node status of
```

Comment:
What about
> Excluded contains the per node status of devices that were identified as candidates but could not be included in the volume group due to filtering

In particular I think we don't need `"It can be consulted in case the device is not picked up as a device for LVM to use"`